### PR TITLE
Windows: Factor out syslog and journald

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com
@swernli

This PR is part of the proposal described in issue 10662 to port the docker daemon to Windows. This PR factors out unsupported loggers on Windows from the daemon - specifically syslog and journald.

### CODE REVIEW

**Comment by LK4D4** [comment]
LGTM

File: daemon/logdrivers_windows.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package daemon
```
Comment (cpuguy83) [inline daemon/logdrivers_windows.go]: Don't understand this.
Don't logdrivers_linux and logdrivers_windows need build tags?

Comment (cpuguy83) [inline daemon/logdrivers_windows.go]: I guess it's not going to hurt anything since it's just importing the same thing again... nevermind.

Comment (LK4D4) [inline daemon/logdrivers_windows.go]: Filename postfixes can be used instead of tags.

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**